responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN conda env create -p $AZUREML_CONDA_ENVIRONMENT_PATH -f conda_dependencies.ya
     conda run -p $AZUREML_CONDA_ENVIRONMENT_PATH pip cache purge && \
     conda clean -a -y
 
-RUN pip install --pre 'azure-ai-ml'
+RUN pip install --pre 'azure-ai-ml' 'azure-storage-blob<=12.13.0' 'numpy<1.24.0'
```

Comment:
Can we add the numpy and azure-storage-blob to conda dependencies? Seems like lots of dependencies in one line. 

Also does the pre flag apply to the last two dependencies?